### ISSUE TITLE
ucf101_testing.sh accuracy problem

### ISSUE THREAD

**Issue by ofeyliu** [facebook/C3D#376]
Hello! Thanks for your C3D work! 

I met a problem when I tried to run C3D-v1.1/examples/c3d_ucf101_finetuning/ucf101_testing.sh. I had downloaded c3d_resnet18_ucf101_r2_iter_90000.caffemodel weights but my accuracy was low as belows:
![](https://user-images.githubusercontent.com/19247763/38307274-c40db71e-3845-11e8-9628-998baeeddd75.jpg)

Could you tell me what reason may cause this problem? 
Thank you!

**Comment by liulinyi007** [comment]
I met the same problems. The loss is between 3~4, and the top-1 is around 0.15, while top-5 is 0.35. Do you solve it?

**Comment by liulinyi007** [comment]
Thanks very much. I'll try this immediately
Another question is that I finetuned train_resnet18_r2.prototxt on c3d_resnet18_sports1m_r2_iter_2800000.caffemodel and met the same problem.
Did you do it before?

**Comment by ofeyliu** [comment]
Sorry late to reply! I haven't met this problem. But I wonder there may exist some configure problem if you don't revise the code. The file I downloaded from dropbox, I think, is different in the repository folder.
By the way, recently I choose  [Pytorch version Res3D](https://github.com/kenshohara/video-classification-3d-cnn-pytorch) from cvpr2018, May this help you.
